.github/settings.yml

```diff
@@ -15,14 +15,14 @@ repository:
   # By changing this field, you rename the repository.
 
   # Uncomment this name property and set the name to the current repo name.
-  # name: ""
+  name: "terraform-ibm-mq-operator"
 
   # The description is displayed under the repository name on the
   # organization page and in the 'About' section of the repository.
 
   # Uncomment this description property
   # and update the description to the current repo description.
-  # description: ""
+  description: "This module installs an IBM MQ operator on your Red Hat OpenShift Container Platform on VPC landing Zone."
```

Comment:
The module itself deploys on any cluster, its just the extension that will only support "Red Hat OpenShift Container Platform on VPC landing Zone" so suggest updating this